### PR TITLE
Support Swift 2016-07-25

### LINKED ISSUE #107: Support 2016-07-25 Toolchain
The latest Xcode beta changes some methods into properties, such as `URLSessionConfiguration.default()` is now a class property `URLSessionConfiguration.default`.

### PR DESCRIPTION
**Pull request by rhyshort**
## What

Support Swift 2016-07-25  Snapshot.

## Issues

Resolves #107

### CODE REVIEW

File: Source/SessionCookieInterceptor.swift [hunk 1]
```diff
@@ -139,7 +139,7 @@ public class SessionCookieInterceptor: HTTPInterceptor
 
                         // Only check for ok:true, https://issues.apache.org/jira/browse/COUCHDB-1356
                         // means we cannot check that the name returned is the one we sent.
-                        guard let ok = jsonResponse["ok"] as? NSNumber where
+                        guard let ok = jsonResponse["ok"] as? NSNumber   where
```
Comment (ricellis) [inline Source/SessionCookieInterceptor.swift]: what is the extra space for?

Comment (ricellis) [inline Source/SessionCookieInterceptor.swift]: Actually the build failure suggests maybe this was intended to be a `,`

File: Tests/SwiftCloudant/QueryViewTests.swift [hunk 2]
```diff
@@ -53,13 +53,13 @@ public class QueryViewTests: XCTestCase {
     func testViewGeneratesCorrectRequestUsingStartAndEndKeys() throws {
         let view = QueryViewOperation(name: "view1", designDocumentID: "ddoc", databaseName: self.dbName!,
                                       descending: true,
+                                      startKey: "startKey",
```
Comment (ricellis) [inline Tests/SwiftCloudant/QueryViewTests.swift]: did the new snapshot necessitate re-ordering of parameters?

Comment (rhyshort) [inline Tests/SwiftCloudant/QueryViewTests.swift]: Yeah the compiler now enforces ordering.

File: Source/SessionCookieInterceptor.swift (hunk 1 above)
Comment (rhyshort) [inline Source/SessionCookieInterceptor.swift]: I keep getting confused with this one, I get a warning with the `,` saying I should use `where` and `where` results in error.

Comment (rhyshort) [inline Source/SessionCookieInterceptor.swift]: screen shot 2016-07-29 at 14 58 48

Comment (rhyshort) [inline Source/SessionCookieInterceptor.swift]: The third build seems to have resolved the compiler error I was getting, fixed in bd5b673

**Comment by ricellis** [comment]
+1

**Pull request merged**